setup.py

```diff
@@ -244,13 +244,13 @@ def run(self):
     
     # Code modified from this example:
     # http://threebean.org/blog/2011/06/06/installing-from-pip-inside-python-or-a-simple-pip-api/
-    import pip.commands.install
+    from pip.commands.install import InstallCommand
```

Comment:
@cpauya I'm not sure if this solves the issue, could you recreate it?

Reporter was reporting that 'dict' object (`commands`) has no attribute `install`.